deployer/auth.py

```diff
@@ -63,6 +65,15 @@ def _get_callback_url_list(self, domains):
 
         return [f'https://{domains}/hub/oauth_callback']
 
+    def _get_allowed_logout_url_list(self, domains):
+        if isinstance(domains, list):
+            logout_urls = []
+            for domain in domains:
+                logout_urls.append(f'https://{domain}/hub/')
+            return logout_urls
+
+        return [f'https://{domains}/hub/']
```

Comment:
Why the `/hub` prefix? Can you add a comment about what is being authorized here?

Reply:
Right, we don't actually need the `/hub` part. Fixed and commented :D